.github/workflows/Push.yaml

```
@@ -0,0 +1,137 @@
+name: libsovtoken Push
+
+on:
+  push:
+    branches:
+      - stable
```

Allow GHA to be run from Gitpod Branch
```suggestion
      - stable
      - Gitpod
```